src/WebJobs.Script.WebHost/FileMonitoringService.cs

```diff
@@ -237,23 +239,25 @@ private async Task ScheduleRestartAsync(bool shutdown)
             }
             else
             {
-                await _restart();
+                _restart();
```

Comment:
Why not awaiting this anymore?

Reply:
Because we didn't await restarts to begin with.

Reply:
Sorry, I take that back. Yes, await here seems to be fine